projects/plugins/jetpack/extensions/shared/components/stripe-nudge/index.jsx

```diff
@@ -17,7 +17,7 @@ export const StripeNudge = ( { blockName } ) => {
 			block: blockName,
 		} );
 
-	if ( store.getShouldUpgrade() || ! stripeConnectUrl ) {
```

Comment:
I think we'll need to update the tests accordingly before we can merge:
https://github.com/Automattic/jetpack/actions/runs/4928397209/jobs/8806728553?pr=30432